reconcile/utils/sharding.py

```diff
@@ -13,7 +13,8 @@ def is_in_shard(value):
     if SHARDS == 1:
         return True
 
-    value_md5 = hashlib.md5(value.encode())
+    h = hashlib.new('md5', usedforsecurity=False)
```

Comment:
I think that this change could break the method?

```python
>>> value = 'test'
>>> import hashlib
>>> h = hashlib.new('md5', usedforsecurity=False)
>>> value_md5 = h.update(value.encode())
>>> type(value_md5)
<class 'NoneType'>
>>> value_hex = value_md5.hexdigest()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'hexdigest'
```

I think it might be worthwhile to add a quick unit test test or two for this function?

Reply:
Also, just as an aside, it looks like pylint was complaining about unknown kwargs before. You could keep the `hashlib.md5` call if you just add a comment like:

```python
value = doing_something_that_pylint_doesnt_like()  # pylint: disable=<check-name>
```

Reply:
Yes you are 100% right\